src/dataviews/category-dataview-model.js

```diff
@@ -44,6 +45,7 @@ module.exports = DataviewModelBase.extend({
     this._searchModel = new SearchModel();
 
     this.on('change:column change:aggregation change:aggregation_column', this._reloadMap, this);
+    this.on('change:prefix change:suffix', this._reloadMap, this);
```

Comment:
They're included in the data sent to Windshaft, so judging from that I'd say yes, but @rochoa or @alonsogarciapablo can perhaps confirm?